app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md

```diff
@@ -0,0 +1,146 @@
+
+Some candidates hesitated when it came to entering the start and end year and month for some of their early roles, and had to spend some time thinking about this. When asked, they suggested that they weren’t completely certain, but were able to either work it out or enter a reasonable guess by anchoring their early jobs to other significant events in their lives.
+
+One candidate consistently entered their employers’ names within the Job title field, and then had to correct themselves. Other candidates entered the employer names first, and then returned to the job title field later having had to give it more thought.
+
+Some candidates suggested that they would be able to get the relevant information from their CV, but no candidates had a CV that included their complete work history.
+
+Not all candidates responded to the prompts to enter explanations of breaks within their work history. Those that did were able to explain their break, eg by mentioning unemployment or raising children, with no stress and using only a brief explanation.
+
+## What we changed
+
+We moved the section back underneath the Work experience section, and reinstated the Unpaid experience section.
+
+We reordered the Add job form, to make the ‘Name of employer’ the first field, as this was often what candidates recalled first.
+
+We renamed ‘Job title’ to ‘Role’, as candidates may not remember their exact job title (or even have had one), and so ‘Role’ implies they can use a short description instead.
+
+We added some hint text to ‘Role’ saying “If you had more than one role with this employer, enter your last role.” as feedback from providers suggested that listing out each promotion within a employment was unnecessary.
+
+We added a checkbox labelled “I’m not sure of the exact month or year I started” and ”I’m not sure of the exact month or year I left” for candidates who struggled with the exact months and years.
+
+We renamed “Did this job involve working in a school or with children?” to “Did you use skills relevant to teaching in this job?” as providers indicated that they were most interested in _relevant_ work history, which could encompass a wider variety of jobs.
+
+We added the hint text “You can go into more detail about this in your personal statement.” to the ”Did this job involve working in a school or with children?” question to encourage candidates to give more details about relevant jobs in their personal statement.
+
+On the review page, we simplified the design, moving away from a table and just using headings with horizontal lines between jobs, enclosed within a bordered section. This removed the need to include column headings and a table caption, and addressed accessibility issues with having used a row of merged cells for the break in history prompts.
```

Comment:
```suggestion
On the review page, we moved away from a table and just using headings with horizontal lines between jobs, enclosed within a bordered section. This removed the need to include column headings and a table caption, and addressed accessibility issues with having used a row of merged cells for the break in history prompts.
```

Reply:
Good call, although you made the rest of the sentence ungrammatical... I think this would work (also not sure about 'moved' any more):

> On the review page, we switched from a table to headings with horizontal lines between jobs, enclosed within a bordered section.